_posts/2014-08-18-nutella-baguette.md

```diff
@@ -17,4 +17,4 @@ categories: desert
 
 Spread a healthy amount of Nutella on to a baguette, and place as much peanut butter as you feel acceptable on to the other half of the baguette.
 
-Slice banana into `1/2cm` slices and place in a row on the baguette and combine two halves together and take your first bite into bliss.
+Extra layer of nutella instead of banana, please
```

Comment:
I think this is a valid change expect the word "please" in the end, if you submit the change, I'll be happy to merge :smile: